cpp/CMakeLists.txt

```diff
@@ -29,7 +29,11 @@ if(WIN32)
     set(CMAKE_CXX_FLAGS_RELEASE "/MT /DNDEBUG")
     set(CMAKE_CXX_FLAGS_DEBUG "/MTd /RTC1")
 else()
-    set(CMAKE_CXX_FLAGS "-Wall -Wextra")
+    if(${ARCTICDB_USING_CONDA})
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra")
```

Comment:
This is a shame. Can you create an issue please that details why this is the case just for Conda so we can prioritise resolving in the future.

Reply:
https://github.com/man-group/ArcticDB/issues/420

Reply:
this is mostly because xxHash drags in a lot of errors from withinside xxhash functions.
For the conda build we use xxHash with `XXH_INLINE_ALL` which might be the cause for some of these warnings.